community/ndp/parser-v1/src/main/antlr4/org/neo4j/tankparse/TankV1.g4

```diff
@@ -0,0 +1,84 @@
+/** Temporary pre-parser that handles "tank"-codenamed cypher extensions.
+    Functionality here will be consolidated with the regular cypher parser. */
+
+grammar TankV1;
```

Comment:
Why are we using antlr here? We use parboiled in cypher, so it would be good to do the same here also. That would save an additional build step (the lack of this currently prevents compilation in intellij).